### ISSUE TITLE
[Frontend] Fix FormLayout's behaviour in Low Resolution Screens

### ISSUE THREAD

**Issue by erimerkin** [bounswe/bounswe2022group7#422]
In recent [Pull Request #417 ](https://github.com/bounswe/bounswe2022group7/pull/417) it came to the light that fixed width and margins are resulting in a non responsive UI for lower resolution and mobile screens. Since the FormLayout component is also setup in the same way, we need to modify them to conform lower resolution screens.

### Problem Description:

**Description:** Pages don't fit on mobile
- Login Page
<img width="394" alt="resim" src="https://user-images.githubusercontent.com/26660856/202846322-a589b769-5075-4d7e-9999-e9db160cde6b.png">

- Sign-up Page

<img width="394" alt="resim" src="https://user-images.githubusercontent.com/26660856/202846439-22c99a64-8e69-4160-8634-48b62baa9c9f.png">

**Environment**: Website on a mobile & low resolution screens
**Pages:** /auth/login & /auth/signup

**Related Part In the code:**

[FormLayout.js](https://github.com/bounswe/bounswe2022group7/blob/develop/frontend/src/layouts/FormLayout.js)

```JavaScript
const styles = {
  main: {
    width: 'auto',
    display: 'block', // Fix IE 11 issue.
    marginLeft: 100,
    marginRight: 100,
  },
  paper: {
    marginTop: 10,
    paddingBottom: 20,
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center',
    p: 2,
    width: 500,
    marginLeft: "auto",
    marginRight: "auto",
  },
};
```

**Deadline: 23.59, 2022-11-21**
**Reviewer: @CahidArda**

**Comment by erimerkin** [comment]
The bug is fixed and details can be viewed in #429. After the PR is merged this issue will be completed.

**Comment by CahidArda** [comment]
Associated PR is merged. Closing the issue.
